config/gocdb_schema.xml

```diff
@@ -243,7 +243,7 @@
 	<field>
 	    <fname>TITLE</fname>
 	    <length>255</length>
-	    <regex>/^[a-zA-Z\s\-']+$/</regex>
+	    <regex>/^[a-zA-Z\s\-']*$/</regex>
```

Comment:
Are there any titles we have that match on `\s` or `\-`?

Reply:
No

Reply:
Fair enough. Was partly wondering about the contents of the database, and if there were any potential future ones that might.

Reply:
Then we can change it when we add the future ones. This is more permissive than what we had before.

Reply:
Sure, which is why I approved this PR and just added a comment.